tools/cli/internal/openapi/oasdiff_test.go

```diff
@@ -559,6 +559,230 @@ func Test_MergeResponses(t *testing.T) {
 	}
 }
 
+func Test_MergeSchemas(t *testing.T) {
+	testCases := []struct {
+		inputBase     *load.SpecInfo
+		inputExternal *load.SpecInfo
+		diff          *diff.Diff
+		name          string
+		error         bool
+	}{
+		{
+			name: "SuccessfulMergeWithEmptySchemas",
+			diff: &diff.Diff{},
+			inputBase: &load.SpecInfo{
+				Url: "base",
+				Spec: &openapi3.T{
+					Components: &openapi3.Components{
+						Schemas: nil,
+					},
+				},
+				Version: "3.0.1",
+			},
+			inputExternal: &load.SpecInfo{
+				Url: "external",
+				Spec: &openapi3.T{
+					Components: &openapi3.Components{
+						Schemas: nil,
+					},
+				},
+				Version: "3.0.1",
+			},
+			error: false,
```

Comment:
```suggestion
			wantErr: require.NoError,
```
on how to use `wantErr`

Reply:
updated